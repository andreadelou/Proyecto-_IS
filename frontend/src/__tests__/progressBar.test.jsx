import React from 'react';
const { render } = require("@testing-library/react")
const { default: ProgressBar } = require("../components/ProgressBar")

describe('Tests para <ProgressBar/>', () => {
	test('Renders a progress bar', () => {
		const { container } = render(<ProgressBar value={0} max={100} color={"blue"} />)
		expect(container).toMatchSnapshot()
	})
})