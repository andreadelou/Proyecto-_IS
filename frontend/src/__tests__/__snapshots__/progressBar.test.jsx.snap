// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests para <ProgressBar/> Renders a progress bar 1`] = `
<div>
  <div
    class="sc-bczRLJ ePzFnv"
    color="blue"
    width="680px"
  >
    <progress
      max="100"
      value="0"
    />
  </div>
</div>
`;
